infra/terraform-dev.tfvars

```diff
@@ -20,3 +20,5 @@ signer_configs = [
   }
 ]
 jwt_signature_pk_url = "https://www.googleapis.com/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com"
+otlp_endpoint        = "https://otel.dev.api.pagoda.co:443/v1/traces"
+opentelemetry_level  = "debug"
```

Comment:
Hm... should it be info? We are not very strict with levels though :/

Reply:
I was thinking to use info for mainnet/testnet and debug for dev just so we have more information there